components/renku_data_services/project/blueprints.py

```diff
@@ -239,7 +242,10 @@ async def _get_all_members(_: Request, user: base_models.APIUser, project_id: st
                 user_id = member.user_id
                 user_info = await self.user_repo.get_user(id=user_id)
                 if not user_info:
-                    raise errors.MissingResourceError(message=f"The user with ID {user_id} cannot be found.")
+                    raise errors.MissingResourceError(
+                        message=f"The user with ID {
+                                                      user_id} cannot be found."
```

Comment:
issue: not sure what's going on with the formatting here :slightly_smiling_face:

Reply:
I think it's `black` formatting from a vscode extension. 🙃